web/template/partial/course/manage/lecture-management-card.gohtml

```diff
@@ -323,12 +302,42 @@
                         </div>
                     {{end}}
                 </div>
-                <template x-if="lecture.uiEditMode > 0"> <!-- only render if in edit mode to avoid network calls -->
-                    <div class="mt-3">
-                        {{template "editvideosections"}}
-                    </div>
-                </template>
-            </div>
+            </article>
+
+            <template x-if="lecture.hasAttachments()">
+                <article>
+                    <h6 class="text-sm text-5 font-light border-b dark:border-gray-800">Attachments</h6>
+                    <ul class="list-disc grid py-2 ml-6">
+                        <template x-for="file in lecture.files">
+                            <template x-if="file.fileType === 2">
+                                <li>
+                                    <section class="flex items-center">
+                                    <span class="text-xs font-semibold text-3 my-auto"
+                                          x-text="file.friendlyName"></span>
+                                        <button class="px-3"
+                                                @click="lecture.deleteFile(file.id)">
+                                            <i class="fa fa-xmark"></i>
+                                        </button>
+                                    </section>
+                                </li>
+                            </template>
+                        </template>
+                    </ul>
+                </article>
+            </template>
+
+            <template x-if="lecture.uiEditMode > 0"> <!-- only render if in edit mode to avoid network calls -->
+                {{template "editvideosections"}}
+            </template>
+
+            <article>
+                <h6 class="text-sm text-5 font-light border-b dark:border-gray-800">Live Chat</h6>
```

Comment:
I like this way better now. 👍
I have an idea how we could make this look even better by having a slider element like this in the left corner of the edit section on the same layer the discard and the save button are.
We could also use this new UI element then in multiple places. I think it can be a bit nicer as it easier to grab and more visible, especially if there is only one toggle.

What do you think of this?

Reply:
Example for such a toggle button: https://flowbite.com/docs/forms/toggle/

Reply:
<img width="695" alt="image" src="https://user-images.githubusercontent.com/29633518/232744770-124e42b0-974c-4103-aa63-268bb9a5432b.png">